src/web/mjs/connectors/Hocvientruyentranh.mjs

```diff
@@ -0,0 +1,59 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class HocVienTruyenTranh extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'hocvientruyentranh';
+        super.label = 'Hoc Vien Truyen Tranh';
+        this.tags = [ 'manga', 'vietnamese', 'webtoon' ];
+        this.url = 'https://hocvientruyentranh.net';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.__info h3.__name');
+        let id = uri.pathname;
+        let title = data[0].textContent;
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let request = new Request(new URL('/truyen/all', this.url), this.requestOptions);
+        let pages = await this.fetchDOM(request, 'ul.pagination li:nth-last-child(2) > a');
+        pages = Number(pages[0].text);
+
```

Comment:
TIP:
During development you may add the following lines to check if manga list loading basically works and the correct page number is detected:
```javascript
console.log('Page Count:', pages);
pages = 5;
```
Don't forget to remove these lines before making the PR

Reply:
Okay.. Thanks.. I always learn so many things in this.. I’m blessed xD